packages/design-system/src/components/Table/TableCaption.jsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+
+export const TableCaption = ({ children, className, scrollableAlert, ...attributeOptions }) => {
+  const classes = classNames('ds-c-table__caption', className);
+
+  return (
+    <caption className={classes} {...attributeOptions}>
+      {children}
+      {scrollableAlert}
+    </caption>
+  );
+};
+
+TableCaption.propTypes = {
+  /**
+   * The table caption contents.
+   */
+  children: PropTypes.node,
+  /**
+   * Additional classes to be added to the caption element.
+   */
+  className: PropTypes.string,
+  /**
+   * Additional text or content to display when the horizontal scrollbar is visible to give the user notice of the scroll behavior.
+   */
+  scrollableAlert: PropTypes.node,
```

Comment:
Based off [this discussion](https://github.com/CMSgov/design-system/pull/706#discussion_r440244955)

I think this should be required, this way scrollable tables will always have a caption. It already has a default value and users can always override it.

Reply:
@nichia This is still relevant I believe